crates/hir_ty/src/lib.rs

```diff
@@ -566,7 +566,7 @@ mod tests {
 
                     stmt_str
                 }
-                hir::Stmt::ExprStmt {
+                hir::Stmt::Expr {
```

Comment:
**Performance Issue**: The change from 'hir::Stmt::ExprStmt' to 'hir::Stmt::Expr' might introduce inconsistencies in how expressions are handled, potentially affecting performance if the semantics of expression evaluation or side effects are altered. <br> **Fix**: Review the implementation of 'hir::Stmt::Expr' to ensure it aligns with the expected performance and side effects of the previous 'ExprStmt'. If discrepancies are found, consider revising the handling of 'Expr' to match the performance characteristics of 'ExprStmt'. <br> **Code Suggestion**: 
 ```
 -                hir::Stmt::ExprStmt {
 +                hir::Stmt::ExprStmt {
 ```